### DISCUSSION TITLE
Future of the DDEV-Local project now that the DDEV company has shut down

### DISCUSSION THREAD

**Discussion by rooflack** [ddev/ddev#3005] [General]
Hi there,
I'd like to know if there were any updates on what the future of DDEV-Local will be, after the DDEV-Live platform is shut down?

I haven't heard anything after the notice that was published a few weeks ago on the now offline ddev.com Website, which said further details would come about DDEV-Local specifically. (Although, to be honest, I haven't really followed the news closely, so I may have missed the announcement if there were any posted somewhere.)

Anyway, just wondering if anything will change for DDEV-Local, how the end of DDEV-Live may affect future maintenance, and how the community could help if need be.

Thanks a lot to Randy and other maintainers for your great work on this this awesome tool!
Cheers,
   GM

**Comment by rfay** [comment]
Hi @rooflack - Thanks for asking here. This has been widely discussed on Twitter and Slack, but not in the issue queue, so I'm glad you brought it up here.

DDEV-Local will continue, I'll continue to support it. Already development has continued as normal, and there have been two minor releases since I was laid off.

The former investors have not yet decided everything though, so it remains possible that there will be a name change or fork required, so we'll see about that. 

So keep using it with confidence, and we'll all make it through whatever comes next.

A couple of companies and organizations have already stepped up to see if they could offer financial support, so that should help as well as things sort out.

**Reply by rooflack** [reply]
Hi @rfay,

Thanks for the (always) quick response! Thank you for letting all of us not on Twitter or Slack know about this. (Is this the Drupal or Typo3 Slacks you're referring to, or is there a dedicated one?)

Of course I'm super glad to hear that the ball is still rolling and the project will continue, and I really hope funding will get sorted out soon :crossed_fingers:

**Reply by rfay** [reply]
Yes, Drupal and TYPO3 slacks, there is not a dedicated slack.

**Reply by andy-blum** [reply]
@rfay went looking for fallout this morning from the DRUD announcement and found this. Is DDEV accepting github sponsors at this time?

**Reply by rfay** [reply]
No, things are still sorting out at this point.. One kind agency that uses ddev is supporting me directly by supporting specific feature development, and there will be other options in the future. Regardless... DDEV remains fully supported and will be in the future.

**Reply by rfay** [reply]
DDEV's future is assured in many ways. Note that Fruition has [acquired the assets of the DDEV company](https://fruition.net/about/blog/fruition-acquires-ddev/) and committed to keep DDEV open-source and to support it. 

Listen to [Mediacurrent podcast](https://www.mediacurrent.com/blog/future-ddev-bright-discussion-randy-fay/) with Randy Fay explaining the bright future of DDEV.
